src/test/java/net/datafaker/providers/base/AbstractBasicProviderTest.java

```diff
@@ -0,0 +1,58 @@
+package net.datafaker.providers.base;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Supplier;
+
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public abstract class AbstractBasicProviderTest<T extends BaseFaker> extends BaseFakerTest<BaseFaker> {
+    @SuppressWarnings("unchecked")
+    protected List<String> getBaseList(String key) {
+        return (List<String>) faker.fakeValuesService().fetchObject(key, faker.getContext());
+    }
+
+    /**
+     * Ensures that the supplied method actually returns a value in the expected list.
+     * 
+     * @param method the method from which the test value will be obtained.
+     * @param key the YAML key from which the list of actual items should be retrieved.
+     */
```

Comment:
I suspect the javadoc comment is obsolete now

Reply:
I re-jigged this. Maybe it's unnecessary? Lemme know if you'd rather I just remove it. (I guess in reality there are now plenty of examples for people to refer to.)

Reply:
yes.. i think it could be simply dropped

Reply:
Done